lib/App/MintTag.pm

```diff
@@ -493,6 +495,42 @@ sub _maybe_delete_source_branches ($self, $step) {
   }
 }
 
+sub _maybe_cleanup_tags ($self, $step) {
+  return unless ($step->push_tag_to && $step->cleanup_tag_days);
+
+  my $remote = $step->push_tag_to;
+  my $prefix = $step->tag_prefix;
+
+  try {
+    my $tag_spec = "refs/tags/${prefix}*";
+    my $tag_list = run_git('for-each-ref', "--format=%(refname:short)\t%(creatordate:unix)", $tag_spec);
+
+    my @tags = map {; my ($tag, $ts) = split /\t/ ; { ts => $ts, 'tag' => $tag}  }
+    split /\r?\n/, $tag_list;
```

Comment:
I _think_ this `\r?` is harmless but should not be needed.  On unixy operating systems, the lines should be split by `\n` and that's what we nearly always use.  If you're not shooting for Windows compatibility, I'm basically positive `\n` is fine.  Generally, on Windows, Perl does some nonsense to give you thinks split by `\n` unless you say otherwise.  I'm not sure how it applies to whatever `run_git` does, though.

Anyway, the short version is:  We "always" split program output by `/\n/` so doing otherwise deserves a comment if it's not wrong. 🙂

Reply:
I copied this code from somewhere else where it had the `\r`, so I blame whoever wrote that code!